src/components/uploader/css/Uploader.css

```diff
@@ -241,7 +241,7 @@ tr.fl-uploader-file-state-uploaded button:before {
     display: block;
     overflow: hidden;
     height: 2em;
-    width: 7em;
+    width: 9em;
```

Comment:
I observed bit more width in Chrome when button size is 9em, because of different renderer in Chrome and Firefox 
But 9em seems to be perfect for Firefox